administrate/metrics/overview.md

```diff
@@ -42,6 +42,14 @@ All metrics are stored in [Warp 10]({{< ref "administrate/metrics/warp10.md" >}}
 
 For instance, you can derive metrics over time, do custom aggregations or combine metrics.
 
+### Get alerts
+
+You can set up alerts in Grafana to be notified on your apps and addons consumption. This can be useful to monitor databases capacity or latency.
```

Comment:
```suggestion
You can set up alerts in Grafana to be notified on your apps and add-ons consumption. This can be useful to monitor databases capacity or latency.
```